### ISSUE TITLE
Code editor: the content textarea is unlabelled

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#6260]
When switching Gutenberg to the "Code editor" mode, a title field and a big textarea are the main editable fields in the page. The content textarea misses a label, thus it doesn't have an accessible name. Worth reminding any UI control or form field needs to be labelled...

Strongly recommended: automated tests to check the presence of labels, see #6098

Also, there's no focus style on the textarea (or it's so subtle that I'm not able to see it). I'd recommend to consider to implement a foucs stye and also improve the one for the tilte, which is almost unnoticeable.
